### PR TITLE
LUTECE-2091 : Add CSRF protection to AdminUserJspBean actions

### PR DESCRIPTION
**Pull request by rzara**
Add corresponding tests

### CODE REVIEW

File: src/java/fr/paris/lutece/portal/web/user/AdminUserJspBean.java [hunk 1]
```diff
@@ -845,7 +845,7 @@ public String getModifyAdminUser( HttpServletRequest request ) throws AccessDeni
         model.put( MARK_LOCALE, getLocale( ) );
         model.put( MARK_MAP_LIST_ATTRIBUTE_DEFAULT_VALUES, map );
         model.put( MARK_WORKGROUP_KEY_LIST, AdminWorkgroupService.getUserWorkgroups( getUser( ), getLocale( ) ) );
-        model.put( MARK_MINIMUM_PASSWORD_SIZE, AdminUserService.getIntegerSecurityParameter( AdminUserService.DSKEY_PASSWORD_MINIMUM_LENGTH ) );
```
Comment (jonenst) [inline src/java/fr/paris/lutece/portal/web/user/AdminUserJspBean.java]: Why is this removed ?

Comment (jonenst) [inline src/java/fr/paris/lutece/portal/web/user/AdminUserJspBean.java]: Ah yes the password was moved to another tab I remember now

Comment (rzara) [inline src/java/fr/paris/lutece/portal/web/user/AdminUserJspBean.java]: Because password modification has been moved to its own tab (see ```getModifyUserPassword```). This model variable was a leftover.

**Comment by jonenst** [comment]
Merged thanks !
